### DISCUSSION TITLE
Ejercicio 2.1 TP3

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#105] [Preguntas y respuestas] [answered]
Chicos no entidendo la consigna que dice: Implementar una función que cuente el largo de una cadena de texto de manera segura, La función debe retornar el largo de la cadena o el código de error correspondiente.

quiere que diga cuantos caracteres tiene??

**Comment by martinvilu** [comment]
Si, pero teniendo en cuenta lo que hace a cadenas seguras. La consigna, digamo.

**Reply by martinvilu** [reply, marked as answer]
Si esto contesta a tu pregunta, el botón para apretar sería este:

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/357f9bea-fc59-4ebc-956d-712440e20ad1)
